section4/exercises/classMethods.js

```diff
@@ -27,14 +27,38 @@ Make sure to run the file with node in your command line.
 // Call the `addTopping` method on two burritos, then log the burritos to verify
 // that the proteins have been changed.
 
+
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+  changeProtein() {
+    this.protein = "carnitas";
```

Comment:
This function changes your protein to carnitas every time it is ran. See if you can use an argument to make this a dynamic function. Same with `addTopping` below- how can you make the topping you push variable?